src/http/gateway/resources/gateway.js

```diff
@@ -12,6 +12,21 @@ const Stream = require('readable-stream')
 const { resolver } = require('ipfs-http-response')
 const PathUtils = require('../utils/path')
 
+function detectContenType (ref, chunk) {
+  let fileSignature
+
+  // try to guess the filetype based on the first bytes
+  // note that `file-type` doesn't support svgs, thereore we assume it's a svg if ref looks like it
+  if (!ref.endsWith('.svg')) {
```

Comment:
Is this check necessary?

Reply:
Yes, file-type doesn’t support svgs and might actually return invalid mine types. They state that in their readme.

Reply:
Ah ok, would you mind adding a comment to say that please?

Reply:
Just saw there’s already a comment there sorry!